cfe_internal/enterprise/federation/federation.cf

```diff
@@ -133,18 +133,51 @@ bundle agent federation_manage_files
         perms => default:mog( "600", "root", "root" );
 
     # TODO: Render these from templates:
-    am_feeder::
-      "$(cfengine_enterprise_federation:config.bin_dir)/dump.sh"
+
+    am_feeder|am_transporter|am_superhub::
+      # TODO: Instrument augments
+      "$(cfengine_enterprise_federation:config.bin_dir)/config.sh"
         create => "true",
+        template_method => "mustache",
+        edit_template => "$(this.promise_dirname)/../../../templates/federated_reporting/config.sh.mustache",
         perms => default:mog( "600", "root", "root" );
-    am_transporter::
-      "$(cfengine_enterprise_federation:config.bin_dir)/transport.sh"
+
+      # TODO: Instrument augments
+      "$(cfengine_enterprise_federation:config.bin_dir)/log.sh"
         create => "true",
+        template_method => "mustache",
+        edit_template => "$(this.promise_dirname)/../../../templates/federated_reporting/log.sh.mustache",
+        template_data => '{ "logger": "log", "logger_args": "", "timestamp_cmd": "date --iso-8601=seconds"}',
+        perms => default:mog( "600", "root", "root" );
+
+      "$(cfengine_enterprise_federation:config.bin_dir)/parallel.sh"
+        copy_from => local_dcp( "$(this.promise_dirname)/../../../templates/federated_reporting/parallel.sh" ),
         perms => default:mog( "600", "root", "root" );
+
+    am_feeder::
+      "$(cfengine_enterprise_federation:config.bin_dir)/dump.sh"
+        copy_from => local_dcp( "$(this.promise_dirname)/../../../templates/dump.sh" ),
+        perms => default:mog( "700", "root", "root" );
```

Comment:
templates/dump.sh -> templates/federated_reporting/dump.sh

maybe make a var for the templates/federated_reporting path? a lot of stuff in all these could get typos in them.

Reply:
ack on path

will consider DRY